sc/shared/src/main/scala/sigmastate/eval/GraphBuilding.scala

```diff
@@ -19,6 +20,25 @@ import sigma.serialization.OpCodes
 
 import scala.collection.mutable.ArrayBuffer
 
+
+case class DeserializeRawBytes[V <: SType](bytes: Value[SByteArray], tpe: V) extends NotReadyValue[V] {
```

Comment:
It's not necessary to introduce new operation with opcode for each method, MethodCall is enough and can be translated to MethodCall from GraphBuilding.